omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java

```diff
@@ -238,6 +238,9 @@ public SimpleObject submit(UiSessionContext sessionContext,
         }
 
         Encounter formEncounter = fes.getContext().getMode() == FormEntryContext.Mode.ENTER ? fes.getSubmissionActions().getEncountersToCreate().get(0) : encounter;
+        if (fes.getContext().getMode() == FormEntryContext.Mode.ENTER) {
+        	formEncounter.setEncounterDatetime(new Date());
```

Comment:
why do you have to set this manually here?

Reply:
So that the encounter is created with time. It's currently being created just with the date.